tests/integrationv2/common.py

```diff
@@ -356,8 +358,11 @@ def __str__(self):
 
 
 class KemGroups(object):
-    # oqs_openssl does not support x25519 based KEM groups
+    # oqs_openssl 1.1.1 does not support KEM groups with 128-bit secuirty ECC + Kyber >512
```

Comment:
Did openssl recently add support for x25519 to openssl-1.1.1, or was this statement previously wrong?

Reply:
Also, I just noticed that [OQS just deprecated their fork of OpenSSL 1.1.1](https://github.com/open-quantum-safe/openssl/commit/98ce60974b53ac99b975cf2418aabab7519f1dff). I've created issue #4132 to track upgrading our test dependency to 3.0 and assigned it to myself.